internal/slicer/slicer.go

```diff
@@ -113,6 +109,10 @@ func Run(options *RunOptions) (*Report, error) {
 				if sourcePath == copyrightPath && targetPath == copyrightPath {
 					hasCopyright = true
 				}
+			} else if pathInfo.Kind == setup.GeneratePath {
+				// "generate" implies generating new things in a path. Thus, we
+				// do not want anything from the package for GeneratePath.
+				continue
```

Comment:
To me the question here is, if `chisel-state` does not affect slicer in any way, why are we processing it in `slicer`? I understand we process it so that we can create the parent directory but, what do you think of getting the location in cmd_cut and changing the entry from `generate` to an optional parent directory?